src/server/object_services/md_store.js

```diff
@@ -1389,6 +1414,17 @@ class MDStore {
         });
     }
 
+    // Only for clean up in testing - Don't use unless you are sure!!!!
+    async delete_all_chunks_in_system() {
```

Comment:
I would add `assert(config.test_mode)` to make sure this is not called anywhere else than coretest.